galley/pkg/crd/validation/webhook.go

```diff
@@ -48,6 +48,15 @@ var (
 	runtimeScheme = runtime.NewScheme()
 	codecs        = serializer.NewCodecFactory(runtimeScheme)
 	deserializer  = codecs.UniversalDeserializer()
+
+	// Expect AdmissionRequest to only include these top-level categories
+	validCategories = map[string]bool{
+		"apiVersion": true,
+		"kind":       true,
+		"metadata":   true,
+		"spec":       true,
+		"data":       true,
```

Comment:
When is `data` used with CRD? 

Also, do we need `status` here for future proofing?

Reply:
Including "data" was my attempt at future-proofing ... it is used by Secrets (e.g. _samples/bookinfo/platform/kube/bookinfo-mysql.yaml_ ) but no Istio CRDs use it.

Would you recommend this first version of the check allow both `data` and `status`, just `status`, or neither?